src/som/compiler/MixinBuilder.java

```diff
@@ -165,16 +175,20 @@ public MixinScope getInstanceScope() {
     return instanceScope;
   }
 
-  public MixinBuilder getOuterBuilder() {
-    return outerBuilder;
+  public MixinBuilder getOuterMixinBuilder() {
+    return outerMixinBuilder;
+  }
+
+  public MethodBuilder getOuterMethodBuilder() {
+    return outerMethodBuilder;
   }
```

Comment:
I wonder, shouldn't an outer builder be either a mixin or a method? It can't have both at the same time, or is that also useful?